interpreter/program.go

```diff
@@ -36,7 +36,11 @@ type Program interface {
 	// container is used to resolve type names and identifiers.
 	Container() string
 
-	// GetInstruction returns the instruction at the given expression id.
+	// Return the runtime expression id corresponding to the expression id from
+	// the AST.
+	GetRuntimeExpressionId(exprId int64) int64
+
+	// GetInstruction returns the instruction at the given runtime expression id.
 	GetInstruction(exprId int64) Instruction
```

Comment:
Perhaps rename the exprId arg to runtimeId here?

Reply:
I renamed the exprId in GetInstruction to runtimeId